### DISCUSSION TITLE
Option to keep display on while charging

### DISCUSSION THREAD

**Discussion by CalcProgrammer1** [InfiniTimeOrg/InfiniTime#1627] [Ideas]
### Verification

- [X] I searched for similar feature request and found none was relevant.

### Pitch us your idea!

Keep the screen on while connected to the charger

### Description

It would be nice if there were an option to leave the screen on continuously while the PineTime is connected to external power.  Being able to dock it and have it there as a desk clock while it's charging would be nice.

**Comment by Riksu9000** [comment]
This could be nice, but I would be concerned about the screen and/or the backlight wearing out faster, and I wouldn't be surprised if the components in such a cheap watch are prone to failing sooner.

Does anyone know if this would have a significant impact on the lifespan of the screen or backlight?

**Comment by escoand** [comment]
Is it possible to show some content without backlight? Some smartphone doing this while loading at night.

**Comment by srt54558** [comment]
The display is not OLED and may consume alot of power while charging. And LCD (especially such cheap one) can burn in (for a period only) and that's not nice

**Comment by sammyj85** [comment]
The always-on mode was already added in 1.15.0, so this is all fulfilled, right?

**Reply by Nitrowing** [reply]
It's so dim that's it's barely visible

**Comment by WhyNotHugo** [comment]
I tried the "[always on](https://github.com/InfiniTimeOrg/InfiniTime/pull/1869)" feature recently implemented, but it dramatically reduces battery life (from ~1 month to ~2days).

I'd love to see a "on while charging" feature, so the screen can remain permanently on while charging on a desk, nightstand or similar situation.
